src/data-catalog/DataBrowserDetails.ts

```diff
@@ -80,7 +92,15 @@ const MetadataDetailsComponent = ({ dataObj, name }) => {
         _.map(
           ({ name, email }) =>
             email
-              ? h(Link, { key: _.uniqueId(`${name}-${email}-`), href: `mailto:${email}`, style: { marginTop: 5, display: 'block' } }, [name])
+              ? h(
+                  Link,
```

Comment:
Are the formatting rules different for ts? I'm wondering why this wasn't formatted this way before.

Reply:
I wonder if they are. Or maybe they've changed since this was last touched and it hasn't errored because it only checks touched files. (That's my best guess)